apps/backoffice-v2/src/common/utils/hsl-to-rgb/hsl-to-rgb.ts

```diff
@@ -0,0 +1,24 @@
+export const hslToRgb = ({
```

Comment:
Are we sure this logic is 100% correct? Should we have tests for it? Are there NPM packages that we can use for this logic?

Reply:
I missed the function not being in use. Removed the function.